Makefile

```diff
@@ -35,3 +35,5 @@ release: build
 	    echo '```' ; \
 	done
 
+lint:
+	golangci-lint run
```

Comment:
github UI complains this does not end witha newline

Reply:
done :+1: